metadata-ingestion/src/datahub/utilities/mapping.py

```diff
@@ -186,22 +204,28 @@ def convert_to_aspects(
             )
             aspect_map[Constants.ADD_TAG_OPERATION] = tag_aspect
         if Constants.ADD_OWNER_OPERATION in operation_map:
-            owner_aspect = OwnershipClass(
-                owners=[
+
+            owners: List[OwnerClass] = []
+
+            for x in sorted(
+                operation_map[Constants.ADD_OWNER_OPERATION],
+                key=lambda x: dict(x)["urn"],
+            ):
+                dict_owner_class = dict(x)
```

Comment:
why is this necessary? shouldn't `x` already be a dict?

Reply:
operation_map[Constants.ADD_OWNER_OPERATION] is pointing to set so we do need to convert dict into tuple before storing it, hence x is tuple

Reply:
added extra if condition, now it is dict only